python/graphstorm/model/node_gnn.py

```diff
@@ -218,15 +219,19 @@ def node_mini_batch_predict(model, emb, loader, return_label=False):
             assert len(input_nodes) == 1, "Currently we only support one node type"
             ntype = list(input_nodes.keys())[0]
             in_nodes = input_nodes[ntype]
-            pred = model.decoder.predict(emb[ntype][in_nodes].to(device))
+            print("change here")
```

Comment:
Are the changes in this node_gnn.py related to the evaluation_frequency argument change?

If not, please separate this change from the current PR.

Thanks.

Reply:
My bad. This file is modified by mistake.